websites/V/Vodafone FM/presence.ts

```diff
@@ -0,0 +1,28 @@
+const presence = new Presence({
+		clientId: "1061398473368412210",
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo_512",
+		details: "Listening to music",
+	};
+
+	if (document.querySelector("#play").classList.contains("hidden")) {
+		presenceData.smallImageKey = "play";
+		presenceData.smallImageText = (await strings).play;
+	} else {
+		presenceData.smallImageKey = "pause";
+		presenceData.smallImageText = (await strings).pause;
+	}
+
+	const artist = document.querySelector("#artist_name")?.textContent,
+		song = document.querySelector("#song_name")?.textContent;
+	if (artist && song) presenceData.state = `${artist} - ${song}`;
```

Comment:
Also maybe add covers 
top of file presence.ts:
```ts
function imgPath(path: string, hostname: string) {
	if (!path) return "https://i.imgur.com/w1jwJTh.png";
	if (path.includes(hostname)) return `https://${path.replace("//", "")}`;
	else return `https://${hostname}${path}`;
}
```
Where you want it to show covers:
```ts
imgPath(document.querySelector("#cover")?.getAttribute("src"), location.hostname)
```

Reply:
Had no idea `largeImageKey` could be changed! I've gone ahead and implemented cover support in https://github.com/PreMiD/Presences/pull/7113/commits/cb2831d83049e9c1be25c165075e72b894b42d27 🚀
 
![image](https://user-images.githubusercontent.com/39736248/211643485-a461e820-0768-45e4-8bb1-27c21d53f3c9.png)

I also found a potentially neat design alternative of showing the Vodafone logo instead of play/pause (since it's a radio and pausing is uncommon), but that would be less features so it's up to y'all.

![image](https://user-images.githubusercontent.com/39736248/211644206-9ec0770f-d432-4125-a9a8-72a893e0f83f.png)

Lmk what you think about this and the implementation of album covers.

Reply:
I've also updated the issue description with up to date screenshots should you want more examples.

Reply:
Seems good to me

Reply:
Cover, yes 👍, also use play - pause icons, if no cover could be found use the website's logo.

Maybe if Artist and Song is found set
Details to song
and State to artist

and only show "listening to music" when they aint found.

Reply:
Applied your suggestion in https://github.com/PreMiD/Presences/pull/7113/commits/7f061cac9fcf68c84a02b1ea6c5469b1b3f7fa28. 

The logo (text) is already being used as a fallback, but I can replace or add Vodafone's logo to it, what do you think? Something like:

![image](https://user-images.githubusercontent.com/39736248/215262590-101d89e2-4996-478b-a4b0-41b95ef6f607.png)